src/Tribe/Commerce/PayPal/Main.php

```diff
@@ -1676,8 +1708,18 @@ protected function get_attendees_by_order_id( $order_id, $ticket_id = null ) {
 	 * @return array
 	 */
 	public function get_order_data( $order_id ) {
-		$name       = get_post_meta( $order_id, $this->full_name, true );
-		$email      = get_post_meta( $order_id, $this->email, true );
+		$name  = get_post_meta( $order_id, $this->full_name, true );
+		$email = get_post_meta( $order_id, $this->email, true );
+
+		$order = Tribe__Tickets__Commerce__PayPal__Order::from_attendee_id( $order_id, [
+			'address_name',
+			'payer_email',
+		] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 484e2c56cbb03fe0caef48ee298d84e3</sub>